src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncIteratorTests.cs

```diff
@@ -9426,15 +9432,15 @@ public void AddVariableCleanup_IntLocal()
 
 var values = C.Produce();
 await foreach (int value in values) { }
-System.Console.Write(((int)values.GetType().GetField("<values2>5__2", BindingFlags.NonPublic | BindingFlags.Instance).GetValue(values)));
+System.Console.Write(((int)values.GetType().GetField("<s>5__2", BindingFlags.NonPublic | BindingFlags.Instance).GetValue(values)));
 
 class C
 {
     public static async System.Collections.Generic.IAsyncEnumerable<int> Produce()
     {
-        int values2 = 42;
+        int s = 42;
```

Comment:
>int s = 42; [](http://example.com/codeflow?start=8&length=11)

It doesn't look like there are meaningful changes in this unit-test. Consider reverting.